morello/src/layout.rs

```diff
@@ -1,9 +1,11 @@
 use itertools::Itertools;
-use nonzero::nonzero as nz;
 use serde::{Deserialize, Serialize};
 use smallvec::{smallvec, SmallVec};
 use std::{collections::HashSet, fmt::Display, hash::Hash};
 
+#[cfg(any(debug_assertions, test))]
```

Comment:
Thank you for your time to review!  Currently, `nz!` is used in `assert_no_size_1_packings` and `LayoutArbRankBounds::default()`.  Since the former is under `#[cfg(debug_assertions)]`, this should be `#[cfg(any(debug_assertions, test))]`.